datadog_checks_dev/datadog_checks/dev/tooling/commands/config.py

```diff
@@ -26,6 +26,12 @@ def config():
     pass
 
 
+@config.command(context_settings=CONTEXT_SETTINGS, short_help='Open the config location in your file manager')
+def explore():
+    """Open the config location in your file manager."""
+    click.launch(CONFIG_FILE, locate=True)
```

Comment:
Could there be an option to just show the path in the terminal?

Reply:
`ddev config find` seems like it will work for that :)

Reply:
Yup! `ddev config find`